File: Wells/Source/Views/Graphics/IChartGraphicsView.vb
Imports Wells.ViewBase
Public Interface IChartGraphicsView
    Inherits IGraphicsView

    Function GetYAxisIndex(axisTitle As String) As Integer
    Sub AddAxis(axis As LiveCharts.Wpf.Axis)
    Sub RemoveAxis(axisIndex As Integer)
    Sub ResetZoom()
End Interface

File: Wells/src/Views/Graphics/IChartGraphicsView.cs
namespace Wells.View
{
    public interface IChartGraphicsView : IGraphicsView
    {
        int GetYAxisIndex(string axisTitle);
        void AddAxis(LiveCharts.Wpf.Axis axis);
        void RemoveAxis(int axisIndex);
        void ResetZoom();
    }
}
